src/category_theory/simplex_category.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+
+import order.category.NonemptyFinLinOrd
+import data.finset.sort
+import tactic.linarith
+
+/-! # The simplex category
+
+We construct a skeletal model of the simplex category, with objects `ℕ` and the
+morphism `n ⟶ m` being the monotone maps from `fin (n+1)` to `fin (m+1)`.
+
+We show that this category is equivalent to `NonemptyFinLinOrd`.
+-/
+
+universe variables u
+
+open category_theory
+
+/-- The simplex category:
+* objects are natural numbers `n : ℕ`
+* morphisms are monotone functions `fin (m+1) → fin (n+1)`
```

Comment:
For consistency with the module doc (or the def), perhaps this should say morphisms from `n` to `m` are monotone functions `fin (n+1) -> fin (m+1)` (or the `m/n` switched version)

Reply:
Thanks.